crates/db_schema/src/traits.rs

```diff
@@ -213,3 +213,16 @@ pub trait InternalToCombinedView {
   /// Maps the combined DB row to an enum
   fn map_to_enum(self) -> Option<Self::CombinedView>;
 }
+
+pub trait PageCursorBuilder {
```

Comment:
Should keep name consistent `PaginationCursor`. Either this should be renamed to `PaginationCursorBuilder`, or `PaginationCursor` should be renamed to `PageCursor`.

Reply:
K, I'll rename.